events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -332,8 +332,9 @@
             try {
                 if (triviaURL === "") {
                     triviaURL = SECRETS.trivia_URL;
-                }
-                triviaURL = triviaURL + "category=" + GSHEET_TAB_NAME;
+                } else if (triviaURL.indexOf("category=") === -1) {
+                    triviaURL = triviaURL + "category=" + GSHEET_TAB_NAME;
```

Comment:
why is this necessary?

Reply:
it was adding "category=GSHEET_TAB_NAME" to the end of the URL everytime getQuestion() was called.

Reply:
right - but i don't understand why this new method of appending `"category=" + GSHEET_TAB_NAME` works. is `triviaURL` not actually a URL?

Reply:
our custom google database has multiple tabs and the way you access them is via that appendage.